blazar_dashboard/static/leases/js/calendar/lease_chart.js

```diff
@@ -141,7 +141,10 @@
               })
               return reservationCopy
             })
-            chart.updateOptions({series: filteredReservations})
+            chart.updateOptions({
+              series: filteredReservations,
+              chart: { height: 60 * filteredReservations[0].data.length + 68}
```

Comment:
This 68 "magic value" is probably good to pull out into a variable/constant and name it like ... vpadding? header height? Not sure what it represents.